[2025-02-26T14:21:27.316+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-26T14:21:27.350+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag.second_task scheduled__2022-01-04T00:00:00+00:00 [queued]>
[2025-02-26T14:21:27.371+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag.second_task scheduled__2022-01-04T00:00:00+00:00 [queued]>
[2025-02-26T14:21:27.371+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-02-26T14:21:27.394+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): second_task> on 2022-01-04 00:00:00+00:00
[2025-02-26T14:21:27.409+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=5965) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-26T14:21:27.410+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag', 'second_task', 'scheduled__2022-01-04T00:00:00+00:00', '--job-id', '493', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp4p_xoa5h']
[2025-02-26T14:21:27.411+0000] {standard_task_runner.py:72} INFO - Started process 6005 to run task
[2025-02-26T14:21:27.411+0000] {standard_task_runner.py:105} INFO - Job 493: Subtask second_task
[2025-02-26T14:21:27.483+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag.second_task scheduled__2022-01-04T00:00:00+00:00 [running]> on host 25887b48f6fb
[2025-02-26T14:21:27.614+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='our_first_dag' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2022-01-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-01-04T00:00:00+00:00'
[2025-02-26T14:21:27.616+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-26T14:21:27.616+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-26T14:21:27.617+0000] {logging_mixin.py:190} INFO - Current task name:second_task state:running start_date:2025-02-26 14:21:27.350650+00:00
[2025-02-26T14:21:27.617+0000] {logging_mixin.py:190} INFO - Dag name:our_first_dag and current dag run status:running
[2025-02-26T14:21:27.617+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-26T14:21:27.618+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-26T14:21:27.619+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2025-02-26T14:21:27.640+0000] {subprocess.py:99} INFO - Output:
[2025-02-26T14:21:27.643+0000] {subprocess.py:106} INFO - hey, I am task2 and will be running after task1!
[2025-02-26T14:21:27.644+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-26T14:21:27.686+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-26T14:21:27.687+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=our_first_dag, task_id=second_task, run_id=scheduled__2022-01-04T00:00:00+00:00, execution_date=20220104T000000, start_date=20250226T142127, end_date=20250226T142127
[2025-02-26T14:21:27.726+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-26T14:21:27.726+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-26T14:21:27.727+0000] {logging_mixin.py:190} INFO - Dag name:our_first_dag queued_at:2025-02-26 14:21:22.170456+00:00
[2025-02-26T14:21:27.728+0000] {logging_mixin.py:190} INFO - Task hostname:25887b48f6fb operator:BashOperator
[2025-02-26T14:21:27.746+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-26T14:21:27.759+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
